candi/bashible/common-steps/node-group/050_configure_and_start_docker.sh.tpl

```diff
@@ -22,8 +22,7 @@ _on_docker_config_changed() {
 {{- end }}
 }
 
-mkdir -p /etc/docker
-bb-sync-file /etc/docker/daemon.json - docker-config-changed << "EOF"
+daemon_json="$(cat << "EOF"
```

Comment:
It would be a bit better to use `jq -n` instead of `cat << EOF`.
A matter of taste.

Reply:
jq sorts fields